app/app.go

```diff
@@ -136,6 +136,41 @@ func New(ctx context.Context, opts ...ConfigOption) (*App, error) {
 		apmOpts = append(apmOpts, apmproxy.WithAgentDataBufferSize(size))
 	}
 
+	if verifyCertsString := os.Getenv("ELASTIC_APM_LAMBDA_SERVER_VERIFY_CERT"); verifyCertsString != "" {
```

Comment:
```suggestion
	if verifyCertsString := os.Getenv("ELASTIC_APM_LAMBDA_VERIFY_SERVER_CERT"); verifyCertsString != "" {
```

I think it would be good to keep the env var names the same as we have for Elastic APM agents (e.g. https://www.elastic.co/guide/en/apm/agent/go/current/configuration.html#config-verify-server-cert), but with the extra "_LAMBDA" bit to ensure the config is only picked up by the extension.